src/main/java/com/amazon/ionpathextraction/PathExtractorBuilder.java

```diff
@@ -265,4 +280,14 @@ public PathExtractorBuilder<T> withSearchPath(final List<PathComponent> pathComp
 
         return this;
     }
+
+    /**
+     * Add a search path by its components, with no annotations matching on the top-level-values.
+     * <br>
+     * @see PathExtractorBuilder#withSearchPath(List, BiFunction, String[])
+     */
+    public PathExtractorBuilder<T> withSearchPath(final List<PathComponent> pathComponents,
+                                                  final BiFunction<IonReader, T, Integer> callback) {
+        return withSearchPath(pathComponents, callback, new String[0]);
```

Comment:
We might as well use a constant for `new String[0]`.

Reply:
This was actually a change from the other PR that I accidentally included here. I addressed it in that PR.